### DISCUSSION TITLE
Additional Scheduling options

### DISCUSSION THREAD

**Discussion by tgnelson** [jhuckaby/Cronicle#648] [Ideas]
Is it possible (I know its not with native cron) to implement something that can launch jobs, for example every 3rd Tuesday of each month or 4th Monday, etc.  It would require knowing the full calendar year and translating that to cron.

**Comment by mikeTWC1984** [comment]
Yes, you can do "custom" schedule. Select weekday you need, then if you need 1st, check "days of the month" 1 thru 7, if 2nd then 8 thru 14, and so on.

**Comment by kranskydog** [comment]
Similarly, the ability to specify 'From the end of the month', so for example, you can run something on the last day of the month, or the last friday of the month. See, for example, BYMONTHDAY and BYDAY at https://docs.oracle.com/en/database/oracle/oracle-database/19/arpls/DBMS_SCHEDULER.html#GUID-73622B78-EFF4-4D06-92F5-E358AB2D58F3

**Comment by SejaMarginal** [comment]
Would also good to have @reboot like Cronie.
A plus with @boot @poweroff!

Thanks!

**Comment by jhuckaby** [comment]
Working on these for v2.0.  Not sure about `@reboot`, as that opens a huge can of worms, but will definitely have advanced calendar options like last day of the month, last friday of the month, etc.

**Reply by reset12** [reply]
Hello, 
is the @reboot like option avaiable now ? I need the option, to start my application after reboot of my server.

**Reply by jhuckaby** [reply]
Hello!  Yes, I am very happy to report, the `@reboot` feature, along with MANY other flexible scheduling options, are now available in xyOps (a.k.a Cronicle v2).  It is currently in beta, but going stable very soon.

Check it out: https://github.com/pixlcore/xyops
